themes/default/content/blog/deploy-to-multiple-regions/index.md

```diff
@@ -0,0 +1,759 @@
+dbPassword, err := random.NewRandomPassword(ctx, "dbPassword", &random.RandomPasswordArgs{
+    Length: pulumi.Int(12),
+})
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+
+```csharp
+// Create a random password for the database's admin user.
+var dbPassword = new Random.RandomPassword("dbPassword", new()
+{
+    Length = 12,
+});
+```
+
+{{% /choosable %}}
+
+{{% choosable language yaml %}}
+
+```yaml
+  # Create a random password for the database's admin user.
+  dbPassword:
+    type: random:RandomPassword
+    properties:
+      length: 12
+```
+
+{{% /choosable %}}
+
+Now we're ready to create the providers. We'll create one per region that we will deploy into:
+
+{{% chooser language "typescript,python,go,csharp,yaml" / %}}
+
+{{% choosable language typescript %}}
+
+```typescript
+// Create two AWS providers, one for each region we'll deploy into.
+const euProvider = new aws.Provider("euProvider", { region: "eu-west-2" });
+const usProvider = new aws.Provider("usProvider", { region: "us-east-1" });
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```python
+# Create two AWS providers, one for each region we'll deploy into.
+eu_provider = aws.Provider('euProvider', region = 'eu-west-2')
+us_provider = aws.Provider('usProvider', region = 'us-east-1')
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+
+```go
+// Create two AWS providers, one for each region we'll deploy into.
+euProvider, err := aws.NewProvider(ctx, "euProvider", &aws.ProviderArgs{
+    Region: pulumi.String("eu-west-2"),
+})
+if err != nil {
+    return err
+}
+usProvider, err := aws.NewProvider(ctx, "usProvider", &aws.ProviderArgs{
+    Region: pulumi.String("us-east-1"),
+})
+if err != nil {
+    return err
+}
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+
+```csharp
+// Create two AWS providers, one for each region we'll deploy into.
+var euProvider = new Aws.Provider("euProvider", new()
+{
+    Region = "eu-west-2",
+});
+
+var usProvider = new Aws.Provider("usProvider", new()
+{
+    Region = "us-east-1",
+});
+```
+
+{{% /choosable %}}
+
+{{% choosable language yaml %}}
+
+```yaml
+resources:
+  # Create two AWS providers, one for each region we'll deploy into.
+  euProvider:
+    type: pulumi:providers:aws
+    properties:
+      region: eu-west-2
+  usProvider:
+    type: pulumi:providers:aws
+    properties:
+      region: us-east-1
+```
+
+{{% /choosable %}}
+
+This is very straightforward in that we've hard-coded the regions into the program. Let's pause for a moment to consider the possibilities. We could have made those configurable too, much like our retention period. Even more powerfully, we could use our programming language's expressiveness to create them more dynamically. For instance, we could have looped over something so that the number is entirely dynamic. We've opted to keep the example simple but you're likely to appreciate these advanced capabilities in real-world scenarios.
```

Comment:
same as previously, recommend dropping the last sentence or adjusting it to use a more specific word than `simple`